service/docs/source/geopmaccess.1.rst

```diff
@@ -1,28 +1,181 @@
-.. role:: raw-html-m2r(raw)
-   :format: html
 
-
-geopmaccess(1) -- Access management for the geopm service
+geopmaccess(1) -- Access management for the GEOPM Service
 =========================================================
 
+SYNOPSIS
+--------
+
+
+Read Access List
+~~~~~~~~~~~~~~~~
+
+.. code-block:: none
+
+    geopmaccess [-c] [-g GROUP | -a]
+
+
+Write Access List
+~~~~~~~~~~~~~~~~~
+
+.. code-block:: none
+
+    geopmaccess -w [-c] [-g GROUP] [-n | -s]
+
 
+Remove Access List
+~~~~~~~~~~~~~~~~~~
 
+.. code-block:: none
 
+    geopmaccess -D [-c] [-g GROUP] [-n]
+
+
+
+Get Help or Version
+~~~~~~~~~~~~~~~~~~~
+
+.. code-block:: none
+
+    geopmaccess -h | -v
 
 
 DESCRIPTION
 -----------
 
-Access management for the geopm service.
+The GEOPM service uses the ``/etc/geopm-service`` directory to store
+files that control which signals and controls may be accessed by a
+user through the service.  The purpose of the ``geopmaccess`` command
+line tool is to enable reading and writing of these access list files.
+
+
+Options
+~~~~~~~
+
+*
+  ``-c``, ``--controls``:
+  Command applies to controls not signals.
+
+
+* ``-u``, ``--user``:
```

Comment:
`-u` is missing from the synopsis. Given that there's a `-g GID`/`--group GID` it seems unconventional to have a `-u`/`--user` that doesn't accept UID. Maybe `--default` instead?